docs/management/admin/blocklist.asciidoc

```diff
@@ -2,7 +2,9 @@
 [chapter]
 = Blocklist
 
-The blocklist allows you to prevent specified applications from running on hosts, extending the list of processes that {elastic-defend} considers malicious. This is especially useful for ensuring that known malicious processes aren't accidentally executed by end users. 
+The blocklist allows you to prevent specified applications from running on hosts, extending the list of processes that {elastic-defend} considers malicious. This helps ensure that known malicious processes aren't accidentally executed by end users. 
+
+The blocklist is not intended to generically block applications assumed to be benign. To compare the blocklist with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

Comment:
I think this is responding specifically to some misconfigurations where admins might have tried to use the blocklist to block an app that isn't a security threat per se, but just something that admins didn't want end users to run. But other readers may have the same confusion you had; maybe there's a clearer way to say this?

Reply:
@nastasha-solomon I'm trying out a twist on your suggestion. I'd like this paragraph to start with "the blocklist does NOT do this" as a contrast to the previous para, which is "the blocklist DOES do this," but otherwise I agree with y'all, a little more explanation is needed here.
```suggestion
The blocklist is not intended to broadly block benign applications for non-security reasons; only use it to block potentially harmful applications. To compare the blocklist with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```
(Is "broadly block benign" too much of a tongue twister? 🤪)

Reply:
I can agree with this approach.  

Sidenote: If this were a Letterkenny episode, I'd applaud your admirable application of alliteration. 🥇